examples/src/main/kotlin/io/aexp/bucketlist/examples/prlifetime/ExportPrLifetimeData.kt

```diff
@@ -131,9 +151,17 @@ object ExportPrLifetimeData {
      * Wrapper to bundle a PR with its activity
      */
     data class PrSummary(val pr: PullRequest, val activity: List<PullRequestActivity>) {
-        val duration: Duration
+        val durationSinceStart: Duration
             get() = Duration.between(activity.first().createdAt, activity.last().createdAt)
 
+        val durationSinceLastPRCommitPushed: Duration
+            get() {
+                var mergeTime: ZonedDateTime = activity.filter({ event -> event.action == "MERGED" }).last().createdAt
```

Comment:
I think that you probably don't need to specify the type of these two times, since they could be inferred.  I don't know that we have a standard for this, but it looks like on lines 67-68 we have a similar val that doesn't specify the type. I'm not particularly sold either way.  Any thoughts on which we like better?